installation/cmd/run.sh

```diff
@@ -41,6 +47,7 @@ do
             shift
         ;;
         *)    # unknown option
+            echo "Unknown input parameter: ${1}"
```

Comment:
Every argument that is not a flag we treat as positional and we shouldn't log them. I you want to log invalid flags add another case:
```
--*)
  echo "Unknown flag ${1}"
  exit 1
  ;;
```
Apply to other scripts too.